jobclient/python/docs/source/conf.py

```diff
@@ -0,0 +1,57 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('../..'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Cook Python Client API'
+copyright = '2020, Two Sigma'
+author = 'Two Sigma'
+
+# The full version, including alpha/beta/rc tags
+release = '0.1.0'
```

Comment:
Is it possible to only have this defined once, instead of both here and in the client itself?